applications/CompressiblePotentialFlowApplication/custom_processes/define_embedded_wake_process.cpp

```diff
@@ -84,6 +85,21 @@ void DefineEmbeddedWakeProcess::ComputeDistanceToWake(){
     KRATOS_CATCH("");
 }
 
+void DefineEmbeddedWakeProcess::SetWakeDistancesSignAccordingToNormal(Element& rElement, BoundedVector<double, 3>& rElementalDistances) {
```

Comment:
```suggestion
void DefineEmbeddedWakeProcess::SetWakeDistancesSignAccordingToNormal(
    Element& rElement,
    BoundedVector<double, 3>& rElementalDistances) {
```
Very tiny style guide issue.